packages/components/src/internal/components/gridbar/ExportModal.tsx

```diff
@@ -53,24 +53,35 @@ export const ExportModal: FC<ExportModalProperties> = memo(props => {
                 <Modal.Title>{title}</Modal.Title>
             </Modal.Header>
             <Modal.Body>
-                <div className="export-modal-body">
-                    <ul>
+                    <table className="export-modal-body">
```

Comment:
This table style does not match the mockup Andrew had, is this intentional?

Reply:
Yes, that's intentional.  It seemed more clear to me.